.rubocop.yml

```diff
@@ -8,8 +8,8 @@ AllCops:
     - 'dashboard/scripts/**/*'
     - 'dashboard/db/**/*'
     - 'dashboard/db/*'
-  RunRailsCops: true
   DisplayCopNames: true
+  TargetRubyVersion: 2.2
```

Comment:
Hound made a change to the rubocop defaults for most cops: https://github.com/houndci/hound/commit/acb0cacef60858840ac858b4470eddee67094534

Mostly they re-enabled rules that are on by default in rubocop.  The two rules they disabled were `Metrics/ModuleLength` and `Metrics/AbcSize`.